[2023-10-01 06:10:09,149] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_fourth_dag_v4.first_task 2021-08-26T00:00:00+00:00 [queued]>
[2023-10-01 06:10:09,455] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_fourth_dag_v4.first_task 2021-08-26T00:00:00+00:00 [queued]>
[2023-10-01 06:10:09,468] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2023-10-01 06:10:09,473] {taskinstance.py:1043} INFO - Starting attempt 1 of 6
[2023-10-01 06:10:09,477] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2023-10-01 06:10:09,653] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): first_task> on 2021-08-26T00:00:00+00:00
[2023-10-01 06:10:09,763] {standard_task_runner.py:52} INFO - Started process 4290 to run task
[2023-10-01 06:10:09,853] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'our_fourth_dag_v4', 'first_task', '2021-08-26T00:00:00+00:00', '--job-id', '507', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/our_fourth_dag.py', '--cfg-path', '/tmp/tmplrh4x0by', '--error-file', '/tmp/tmpnubkzm9a']
[2023-10-01 06:10:09,978] {standard_task_runner.py:77} INFO - Job 507: Subtask first_task
[2023-10-01 06:10:11,020] {logging_mixin.py:104} INFO - Running <TaskInstance: our_fourth_dag_v4.first_task 2021-08-26T00:00:00+00:00 [running]> on host d4ed655215d1
[2023-10-01 06:10:12,004] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=svastikkka
AIRFLOW_CTX_DAG_ID=our_fourth_dag_v4
AIRFLOW_CTX_TASK_ID=first_task
AIRFLOW_CTX_EXECUTION_DATE=2021-08-26T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-08-26T00:00:00+00:00
[2023-10-01 06:10:12,026] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2023-10-01 06:10:12,039] {bash.py:158} INFO - Running command: echo hello world, this is the first task!
[2023-10-01 06:10:12,161] {bash.py:169} INFO - Output:
[2023-10-01 06:10:12,193] {bash.py:173} INFO - hello world, this is the first task!
[2023-10-01 06:10:12,200] {bash.py:177} INFO - Command exited with return code 0
[2023-10-01 06:10:12,516] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=our_fourth_dag_v4, task_id=first_task, execution_date=20210826T000000, start_date=20231001T061009, end_date=20231001T061012
[2023-10-01 06:10:13,645] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-01 06:10:13,948] {local_task_job.py:146} INFO - Task exited with return code 0
